engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/meso/LibBenchRunner.java

```diff
@@ -0,0 +1,66 @@
+package org.enso.interpreter.bench.benchmarks.meso;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.Collections;
+import org.graalvm.polyglot.Context;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.ChainedOptionsBuilder;
+import org.openjdk.jmh.runner.options.CommandLineOptionException;
+import org.openjdk.jmh.runner.options.CommandLineOptions;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+// TODO: Move to different project?
+public class LibBenchRunner {
```

Comment:
What is this? How is this supposed to be used? Ideally I want to execute:
```
sbt:runtime> benchOnly VectorOperations
```
and all the `VectorOperations` benchmarks would be executed. I don't see anything like that in `LibBenchRunner`, neithre in `LibBench`.

Reply:
Since f5a9aae you can use:
```bash
sbt:enso> benchOnly VectorOperations
org.enso.interpreter.bench.benchmarks.meso.VectorOperations.newConstant
org.enso.interpreter.bench.benchmarks.meso.VectorOperations.newRandom
org.enso.interpreter.bench.benchmarks.meso.VectorOperations.newVector
org.enso.interpreter.bench.benchmarks.meso.VectorOperations.sum
```
in JMH.

Reply:
This looks like this is now part of the engine space? The VectorOperations are now part of the engine runtime tests.

Are we retiring the old benchmark project and will we need to get every new benchmark (which we clearly should do more of) approved by engine team going forward?